02-models.md

```diff
@@ -97,7 +97,7 @@ and without which people cannot understand how to use commands
 > ## What Happens Next?
 >
 > An example of how solving problems can help people correct broken mental models,
-> consider this problem from Gelman's *[Thinking Physics](http://www.amazon.com/Thinking-Physics-Understandable-Practical-Reality/dp/0935218084/r)*.
+> consider this problem from Lewis Carroll Epstein's *[Thinking Physics](http://www.amazon.com/Thinking-Physics-Understandable-Practical-Reality/dp/0935218084/r)*.
```

Comment:
Thanks for the catch - can you please change it to just "Epstein" ?  We're a surnames-only-in-citations kind of show...